common/bookmarks.cwt

```diff
@@ -69,6 +69,10 @@ bookmark = {
 
             ### The character's characteristics.
             character = {
+                ## cardinality = 0..1
+                ### The character's gender is or is not female, optional.

```

Comment:
For bools (esp. those with default values) please use the following format since I've already established it in most places: `If yes [or not default], something something. Defaults to "no".` Speaking of which, I'll have to remove all those unnecessary quotes.

Reply:
Done.